npm/private/npm_linked_package_direct_info.bzl

```diff
@@ -0,0 +1,14 @@
+"NpmLinkedPackageDirectInfo provider"
+
+NpmLinkedPackageDirectInfo = provider(
+    doc = "Provides a direct linked npm package",
```

Comment:
what is a "direct linked npm package"?

Reply:
Also out of date here. I'll ping you for review once I fix it. Will probably need a new PR

Reply:
Force pushed the same commits and now it seems to be displaying correctly..

Reply:
This is out of date now that the code has been refactored further

Reply:
I dropped the "direct" nomenclature. There is now just a "link" or an "npm_package_store". A link is any package linked to node_modules `node_modules/<package>`. An "npm_package_store" is a package within the virtual store / symlinked node_modules tree `node_modules/.aspect_rules_js`